package.json

```diff
@@ -48,6 +48,13 @@
   "license": "MIT",
   "main": "lib/inversify.js",
   "module": "es/inversify.js",
+  "exports": {
+    ".": {
+      "types": "./lib/inversify.d.ts",
```

Comment:
```suggestion
```
If we go this way, tsc will default to cjs types even when ESM imports are used. Is this line critical to solve the issue?

Reply:
No, I just wasn't entirely sure how tsc will behave. I've removed the line again :)